tests/unit/commands/addon-test.js

```diff
@@ -30,13 +30,31 @@ describe('addon command', function() {
 
   it('doesn\'t allow to create an addon named `ember`', function() {
     return command.validateAndRun(['ember']).then(function() {
-      expect(false, 'should have rejected with an addon name of test');
+      expect(false, 'should have rejected with an addon name of ember');
     })
     .catch(function(error) {
       expect(error.message).to.equal('We currently do not support a name of `ember`.');
     });
   });
 
+  it('doesn\'t allow to create an addon named `Ember`', function() {
+    return command.validateAndRun(['Ember']).then(function() {
+      expect(false, 'should have rejected with an addon name of Ember');
```

Comment:
`expect(false)` is not actually asserting anything, you need to use something like `.to.be.true` at the end. see #5483